### ISSUE TITLE
Mobile: delay dragging widget vs page scroll 

### ISSUE THREAD

**Issue by JulienLecoq** [gridstack/gridstack.js#2781]
## Subject of the issue
Currently, the drag start immediately which can cause annoying drag start on mobile device when all you want is to move on the page.

## Your environment
gridstack: "^10.3.1"
gridstack-angular: "^0.6.0-dev"

**Comment by adumesny** [comment]
Also real apps typically have a subset like a widget header to drag not
entire page. On mobile don't you use 2 fingers to drag page? Are you saying
single touch point comes across and drag widget instead? Don't recall that
issue when I tested it.
sent from phone. excuse any typ-oh!

**Comment by JulienLecoq** [comment]
Yes, single touch point comes across and drag widget instead.

I have an explicit edit mode. I have a grid of photos similarly to Tinder and some others app like so:

<img width="393" alt="grid_on_big_screen" src="https://github.com/user-attachments/assets/3bdf36b6-9179-4214-b89b-34da55f570e2">

On big screen, it's okay since we don't have to scroll, but if you're on a smaller screen, this can get annoying since you have to scroll to see the full grid:

<img width="376" alt="grid_on_small_screen" src="https://github.com/user-attachments/assets/d612df3f-94cd-4bf3-afb0-dcfcc76e0167">

**Comment by JulienLecoq** [comment]
I never used two fingers to scroll on a page.

I would say that this issue is not a big problem since this only occurs on this page. I don't have a grid with drag & drop anywhere else. Also, I could swap the position of the two first big pictures with the grid to temporarily solve this issue.

But I guess it's still a nice feature to have.

**Comment by adumesny** [comment]
Yeah 1 finger to page scroll and move items. I guess the only way to
disembiguate would be to require user holding down on the widget long
enough to turn into move mode similar to phone dashboard editing. Makes
sense for mobile. Feature request if someone wants to donate for it.
sent from phone. excuse any typ-oh!

**Comment by JulienLecoq** [comment]
Yes 👍🏻

**Comment by gitzhangshenglin** [comment]
Have you fixed it yet? After clarifying the editing mode, the page also needs to scroll to display everything. At this time, scrolling will cause the box to be dragged. I hope to receive your reply. Thank you

**Comment by adumesny** [comment]
@gitzhangshenglin as I mentioned above if someone wants to donate for this feature, it could be added.

**Comment by adumesny** [comment]
fixed in next release! don't forget to donate if you find this lib useful
